src/common/StandardPage/StandardPage.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { fireEvent, render } from "@testing-library/react";
+import { Plus as PlusIcon } from "lucide-react";
+import { StandardPage } from "./StandardPage";
+
+const StandardPageProps = {
```

Comment:
How about renaming it to `MockStandardPageProps` ?

Reply:
good idea , i have fixed